x-pack/plugins/remote_clusters/__jest__/client_integration/add/remote_clusters_add.helpers.tsx

```diff
@@ -32,11 +23,12 @@ const testBedConfig = ({ isCloudEnabled }: { isCloudEnabled: boolean }) => {
   };
 };
 
-const initTestBed = (isCloudEnabled: boolean) =>
-  registerTestBed(ComponentWithContext, testBedConfig({ isCloudEnabled }))();
-
-export const setup = async (isCloudEnabled = false) => {
-  const testBed = await initTestBed(isCloudEnabled);
+export const setup = async (httpSetup: HttpSetup, overrides?: Record<string, unknown>) => {
+  const initTestBed = registerTestBed(
+    WithAppDependencies(RemoteClusterAdd, httpSetup, overrides),
+    testBedConfig({ isCloudEnabled: !!overrides?.isCloudEnabled })
```

Comment:
nit: I think you don't need to pass `isCloudEnabled` to `testBedConfig` because in `WithAppDependencies` the value will be replaced by whatever is in `overrides`. To make it simpler, I'd only rely on `overrides` to enable Cloud.